paper-api/src/main/java/io/papermc/paper/configuration/ServerConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package io.papermc.paper.configuration;
+
+public interface ServerConfiguration {
```

Comment:
Probably wanna note here that this class doesn't aim to expose every single config option in existance, only select ones.